Contest_API.md

```diff
@@ -330,6 +330,8 @@ access to.
 | [contest\_thaw](#modifying-contests)       | Control the contest's thaw time
 | [team\_submit](#modifying-submissions)     | Submit as a team
 | [team\_clar](#modifying-clarifications)    | Submit clarifications as a team
+| [judge\_clar](#modifying-clarifications)   | Submit clarifications as a judge
+| [judge\_comment](#modifying-commentary)    | Submit commentary as a judge
```

Comment:
Does it make sense to rename this to `post_comment` or so? Commenting would typically be reserved for judges, and the capability can anyways be role restricted, but I don't think that the semantics or details of this capability would be different if another role (say public) would have it.

Reply:
As above, agreed.

Reply:
PR updated, please re-review.